src/modules/navigator/rtl.cpp

```diff
@@ -508,7 +539,17 @@ void RTL::set_rtl_item()
 			_mission_item.lon = _destination.lon;
 			_mission_item.altitude = loiter_altitude;
 			_mission_item.altitude_is_relative = false;
-			_mission_item.yaw = get_bearing_to_next_waypoint(gpos.lat, gpos.lon, _mission_item.lat, _mission_item.lon);
+
+			if (heading_mode == 0) {
```

Comment:
use an `enum class`

Reply:
Implemented.  Thanks @bresch, feel free to push any changes to naming you might prefer.